.vscode/cSpell.json

```diff
@@ -1,46 +1,64 @@
 // cSpell Settings
 // Contains additional words for our project
 {
-  "version": "0.1",
+  "version": "0.2",
```

Comment:
These changes are cool, but not related to the thing this PR says it does ("Fix broken links").

Please don't do multiple things in a single PR.

Reply:
You are right, sorry. I will revert this to close this PR.